src/features/toilet/Toilet.js

```diff
@@ -131,77 +116,84 @@ function Toilet() {
   }
 
   function onClickWaitingSavior() {
-    navigate("/chats", { toilet_id, chatRoomList });
+    navigate("/chats", {
+      state: {
+        toilet_id,
```

Comment:
[Use useNavigate instead of useHistory](https://github.com/remix-run/react-router/blob/main/docs/upgrading/v5.md#use-usenavigate-instead-of-usehistory)

state가 한개이면 바로 내려주어도 괜찮고, 여러개이면 감싸서 내려줘야 할것 같습니다.